src/main/java/storagesystem/viewcontroller/inventory/DetailedItemViewController.java

```diff
@@ -104,10 +108,11 @@ private void initialize() {
                 itemPageConditionSlider.setValue(2);
                 break;
         }
-        /**
-         * listens to change in amount textArea.
-         * if its a letter it doesn't write anything.
-         */
+
+
+        // Listener for the Amount textarea.
+        // "\\d" represents a value [0-9] and if the value inputed doesnt include one of those
```

Comment:
What is \\d? Unclear comment

Reply:
\d tolkas som [0-9]. lade till det i kommentaren